dpanonymize/__init__.py

```diff
@@ -77,9 +110,11 @@ def get_file_objects_from_phoenix(root_dir: Union[Path, str],
         - BIDS: True if the PHOENIX is in BIDS structure, bool.
     '''
     protected_files = []
+    root_dir = Path(root_dir)
+
     for root, _, files in os.walk(root_dir):
         for file in files:
-            full_path = Path(root) / file
```

Comment:
What was incomplete before that we have to insert `str( )`?

Reply:
The `FileInPhoenixBIDS` and `FileInPhoenix` classes require string inputs now, since we're using regrex to catch names. So I'm making the Path objects into strings before putting into the classes.